.github/workflows/main.yml

```diff
@@ -1,19 +1,45 @@
 ---
 name: "Main Workflow"
+env:
+  DOCKER_USERNAME: sharifwss
+  RELEASE_VERSION: 1.0.0
 on:
-    push:
-        branches:
-            - main
-    pull_request:
+  push:
+    branches:
+      - main
```

Comment:
Shouldn't this be `master` rather than `main`? I think we don't have any `main` branch.

Reply:
Ah, fixed. Thanks!